moped-editor/src/views/projects/projectView/ProjectActivityLogTableMaps.js

```diff
@@ -1,166 +1,4 @@
 export const ProjectActivityLogTableMaps = {
-  moped_project: {
-    label: "Project",
-    fields: {
-      project_uuid: {
-        icon: "",
-        label: "unique ID",
-        data_type: "uuid",
-      },
-      project_name: {
-        icon: "",
-        label: "name",
-        data_type: "text",
-      },
-      project_description: {
-        icon: "",
-        label: "description",
-        data_type: "text",
-      },
-      project_description_public: {
-        icon: "",
-        label: "public description",
-        data_type: "text",
-      },
-      ecapris_subproject_id: {
-        label: "eCAPRIS subproject ID",
-        icon: "",
-        data_type: "text",
-      },
-      project_importance: {
-        icon: "",
-        label: "importance",
-        data_type: "integer",
-      },
-      project_order: {
-        icon: "",
-        label: "order",
-        data_type: "integer",
-      },
-      current_status: {
-        icon: "",
-        label: "current status",
-        data_type: "text",
-      },
-      project_id: {
-        icon: "",
-        label: "ID",
-        data_type: "integer",
-      },
-      timeline_id: {
-        icon: "",
-        label: "timeline",
-        data_type: "integer",
-      },
-      current_phase: {
-        icon: "",
-        label: "phase",
-        data_type: "text",
-      },
-      end_date: {
-        icon: "",
-        label: "end date",
-        data_type: "date",
-      },
-      project_length: {
-        label: "length",
-        icon: "",
-        data_type: "integer",
-      },
-      fiscal_year: {
-        icon: "",
-        label: "fiscal year",
-        data_type: "text",
-      },
-      capitally_funded: {
-        icon: "",
-        label: "capitally funded",
-        data_type: "boolean",
-      },
-      project_priority: {
-        icon: "",
-        label: "priority",
-        data_type: "text",
-      },
-      date_added: {
-        label: "date added",
-        icon: "",
-        data_type: "timestamp with time zone",
-      },
-      added_by: {
-        icon: "",
-        label: "added by",
-        data_type: "integer",
-        lookup: {
-          table: "moped_users",
-          fieldLabel: "user_id",
-          fieldValues: ["first_name", "last_name"],
-        },
-      },
-      is_deleted: {
-        icon: "",
-        label: "soft delete status",
-        data_type: "bool",
-      },
-      milestone_id: {
-        icon: "",
-        label: "milestone ID",
-        data_type: "integer",
-      },
-      status_id: {
-        icon: "",
-        label: "status ID",
-        data_type: "integer",
-      },
-      updated_at: {
-        icon: "",
-        label: "updated timestamp",
-        data_type: "timestamp with time zone",
-      },
-      // deprecated column, but keeping because historical activities depend on it
-      contractor: {
-        icon: "",
-        label: "contractor",
-        data_type: "text",
-      },
```

Comment:
😢 I hope we can find a way to zap this historical stuff out of the codebase. I hope we don't get into a situation where we have reserved words that we can't use in the database as a column because we used it once in the past. Not the end of the world but ... here's an idea we might discuss as a team.

I think it would be neat to add a field to the activity log table named `overloaded_message`, and have it default to `null`. For any record that does get it populated, such as we might on activity log entries that concern depreciated columns and features, we could see that field being not-null in this activity log rendering system and print out whatever message is in it, just raw without formatting. This would allow us to drop support for the special names in the data fields of the activity log.